signac/contrib/project.py

```diff
@@ -1440,7 +1442,9 @@ def reset_statepoint(self, job, new_statepoint):
         details="Use job.update_statepoint() instead.",
     )
     def update_statepoint(self, job, update, overwrite=False):
-        """Update the state point of this job.
+        """Change the state point of this job without affecting existing parameters.
```

Comment:
```suggestion
        """Change the state point of this job without affecting existing parameters while preserving job data.
```

Reply:
There's another small issue here, which is that "without affecting existing parameters" is only true if `overwrite=False`, so that might be misleading.

Reply:
I agree with @vyasr that this suggestion is potentially misleading. The summary line of a docstring must be short. It doesn't have to fully explain the behavior or how various parameters affect its behavior - that's better to put in the description after the first line and the "Parameter" sections.

Reply:
I made a separate sentence to address @vyas's point. I left in "while preserving job data" in both to resolve my confusion and @Charlottez112 's suggestion